libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -780,6 +780,19 @@ bool AP_Camera::get_state(uint8_t instance, camera_state_t& cam_state)
     }
     return backend->get_state(cam_state);
 }
+
+// change camera settings not normally used by autopilot
```

Comment:
```suggestion
#if AP_CAMERA_CHANGE_SETTING_API_ENABLED
// change camera settings not normally used by autopilot
```

Reply:
I edited @peterbarker note above

Reply:
I really think we are trying to be too granular.  This is a tiny feature already hidden behind AP_SCRIPTING.  I'll will go ahead and add it because that's the easiest path to getting this merged.